### DISCUSSION TITLE
yoloV8+sahi

### DISCUSSION THREAD

**Discussion by hushuyin** [obss/sahi#1058] [Q&A] [answered]
What should I do if I want to get Precision and Recall in yoloV8+sahi. Also from sahi.evaluation import get_coco_eval and from sahi.model import Yolov8DetectionModel show that they don't exist, are they changing references or is my import step wrong

**Comment by fcakyon** [comment, marked as answer]
Hello @hushuyin,

You can use this function: https://github.com/obss/sahi/blob/9113f382c9951bad524e3f2948817dad7552eb66/sahi/scripts/coco_evaluation.py#L330

or this cli command: https://github.com/obss/sahi/blob/main/docs/cli.md#coco-evaluate-command-usage
